pkg/controllers/scheduledevent/scheduledeventcontroller.go

```diff
@@ -353,6 +363,9 @@ func (s ScheduledEventController) provisionScheduledEvent(templates *hfv1.Virtua
 					UID:        se.UID,
 				},
 			},
+			Labels: map[string]string{
+				"scheduledevent": se.Name,
```

Comment:
let's namespace this. `scheduledevent.hobbyfarm.io` please

Reply:
Good point!